cluster/images/conformance/conformance-e2e.yaml

```diff
@@ -67,7 +67,7 @@ spec:
     - name: E2E_PARALLEL
       value: "false"
     - name: E2E_VERBOSITY
-      value: "4"
+      value: "8"
```

Comment:
Again, I'm not a fan of doing this by default, it's going to be super noisy.
/hold 
/assign @spiffxp @timothysc